archinstall/lib/installer.py

```diff
@@ -320,18 +316,18 @@ def minimal_installation(self):
 				if 'encrypt' not in self.HOOKS:
 					self.HOOKS.insert(self.HOOKS.index('filesystems'), 'encrypt')
 
-		if not(hasUEFI()):
+		if not (hasUEFI()):
```

Comment:
We could remove the excessive `()` around `hasUEFI()` as well.

Reply:
Resolved in latest commit